### ISSUE TITLE
Bug: Tooltip on attachment is not fully shown if attachment has a long name

### ISSUE THREAD

**Issue by lindy65** [mattermost/focalboard#4433]
Found by QA community contributor @krmh04 on `https://boards-bugbash-76.test.mattermost.cloud/` server, Boards v7.7.0

## Steps to reproduce the behavior

1. Attach a file with a long name to a card
2. Hover over the attachment to display the tooltip
3. Observed: tooltip does not display full file name (see screenshots)

## Expected behavior

Tooltip either shows full file name or is purposefully shown as being truncated

## Screenshots (optional)

![image](https://user-images.githubusercontent.com/13020833/211074836-5b4ecb6f-c3b1-425e-80f8-565c4791ae44.png)

![image](https://user-images.githubusercontent.com/13020833/211075050-aef86baa-545e-4205-82f6-4922486788ff.png)

## Edition and Platform

 - Edition: Boards plugin
 - Version: v7.7.0
 - Browser and OS: Windows/Chrome

**Comment by wuwinson** [comment]
@lindy65 Can you confirm if this is happening due to the horizontal scroll or if it's actually being cutoff without scrolling? May be more of a UX issue. cc @asaadmahmood

**Comment by lindy65** [comment]
Hey @wuwinson - I think you're correct - the user needs to scroll to the right to see the full file name.  I feel this isn't obvious though (it wasn't to me or @krmh04) and @jprusch suggested perhaps adding file attachments vertically instead of horizontally [here on the test server](https://boards-bugbash-76.test.mattermost.cloud/focalboard/pl/6ee8da6d73bw3pb6fp4uqgy3be).

Seems there are still a few UX issues - see screenshots below.

![image](https://user-images.githubusercontent.com/13020833/211486674-1b226843-cfbb-456a-a818-731a41a55b25.png)

![image](https://user-images.githubusercontent.com/13020833/211487105-dd491481-74cc-40a1-9330-b267e4530a85.png)

**Comment by wuwinson** [comment]
@asaadmahmood Thoughts on changing file attachment layout to a vertical view?

**Comment by asaadmahmood** [comment]
@wuwinson @lindy65 The tooltip should appear in full and not be cropped by the scroll.
Not inclined to make them a vertical list, as just a few files (4-5) will occupy a lot real estate and push important data way down.

We should increase the content width like in my proposed card improvements, which will probably make this issue much less jarring.

**Comment by jprusch** [comment]
Sorry, but horizontal scrolling is a pain. Another option would be to generally display all files at the bottom & have a quick link to scroll to the files section. +1 on vertical placement of files occupy a lot of space, but with horizontal placement some files will never be discovered.

**Comment by lindy65** [comment]
Thanks @asaadmahmood @wuwinson - agree with Asaad that a vertical list of attachments will take up a lot of real estate

Just a thought / question in the screenshot below

![image](https://user-images.githubusercontent.com/13020833/211628104-3cfa2e4b-df62-467e-9e15-1742de337c33.png)

Another idea is a horizontal scroll bar something like the screenshot below but horizontal of course

![image](https://user-images.githubusercontent.com/13020833/211628393-8fb84b70-e801-44d5-99b7-899eac9e0bdd.png)

**Comment by asaadmahmood** [comment]
@lindy65 agreed we can have a scrollbar. Though I think it should already be there, for some devices, except those that actually hide the scrollbar until you're scrolling or hover over it.

**Comment by lindy65** [comment]
Hmmm... why do some devices make it difficult to discover things!?! 😄 I don't see a scroll bar on Windows10/Chrome
